fili-wikipedia-example/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM druidio/example-cluster:latest
+LABEL maintainer="https://groups.google.com/forum/#!forum/fili-developers"
+
+# fili github information
+ENV GITHUB_OWNER=yahoo BRANCH=master
+
+RUN apt-get update && apt-get install -y --no-install-recommends git
```

Comment:
@kevinhinterlong After I added ```curl``` to the end of this line, Codacy had some complaints, like 

- either use wget or curl but not both 

- specify a version when you use apt get install

- delete the apt-get lists after installing something

Should I try to fix these issues? 

Codacy also complained about some things that I already fixed, like not specifying a maintainer. Strangely, it didn't have any issues with the last commit.